addons/im_livechat/tests/test_get_operator.py

```diff
@@ -221,3 +223,70 @@ def test_priority_by_number_of_chat_all_operators_exceed_limit(self):
         self._create_chat(livechat_channel, second_operator)
         self._create_chat(livechat_channel, second_operator)
         self.assertEqual(first_operator, livechat_channel._get_operator())
+
+    def test_max_concurent_sessions(self):
+        operator = self._create_operator()
+        livechat_channel = self.env["im_livechat.channel"].create(
+            {
+                "name": "Livechat Channel",
+                "user_ids": [operator.id],
+                "concurent_session": True,
+                "concurent_session_number": 2,
+            }
+        )
+        self._create_chat(livechat_channel, operator)
+        self._create_chat(livechat_channel, operator)
+        self.assertEqual(False, livechat_channel._get_operator())
+
+    def test_max_concurent_sessions_expired(self):
```

Comment:
Misleading name and complex test. Should just test "available_operator_ids":
- operator has no chats, it is available
- create two chats, he's not anymore